doc/nrf/doc_build.rst

```diff
@@ -65,7 +65,7 @@ Complete the following steps to build the documentation output:
 
         .. code-block:: console
 
-           cmake -GNinja -B_build .
+           cmake -GNinja -B _build
```

Comment:
I just tested it - seems that both work.
@tejlmand should we change it or not?

Reply:
Retaining the text as is..